.github/workflows/deploy-to-staging.yml

```diff
@@ -46,20 +46,3 @@ jobs:
             --name  /pya/staging/web/version \
             --value "$IMAGE_TAG" \
             --overwrite
-
-      - name: Trigger infrastructure deployment
```

Comment:
we can add this back in once the deployment GHA is set up in `tax-benefits-backend`